.github/workflows/pr.yml

```diff
@@ -57,6 +57,19 @@ jobs:
           echo "not well formatted sources are found"
           exit 1
         fi
+    -
+      name: Check manifests
+      run: |
+        git reset HEAD --hard
+
+        # Note: fmt is necessary after generate since generated sources will
+        #       fail format check by default.
+        make generate fmt manifests
```

Comment:
it fails with `Error patching crds: yq is required`

Reply:
Ah bummer. That's probably why we didn't include it it in the first place.

Reply:
Added `pip install yq` and it seems to work without problems